tests/python/unittest/test_tvmscript_roundtrip.py

```diff
@@ -3622,5 +3622,14 @@ def test_return_none_no_trailing_type():
     assert "-> None" not in script
 
 
+def test_pow_roundtripable():
```

Comment:
Could you please use the `ir_generator` to make tests consistent in this file?

Reply:
Okay.